### PR TITLE
API Testing Env

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

https://saucedev.atlassian.net/browse/TCD-2086

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/environments.md [hunk 1]
```diff
@@ -7,51 +7,63 @@ description: "Gain a wide range of options to mix and match your test settings w
...
 * An existing API Testing Project and Test. For details on how to create them, see [API Testing Quickstart](/api-testing/quickstart/).
 * Recommended: review [Creating Reusable Variables and Snippets with the Vault](/api-testing/vault/)
 
-## Using Environments
 
-Once a domain is parametrized, you can override a variable, if needed.
+## Creating Environments
 
-To access and create new environments:
+Once a domain is parametrized, you can override a variable if needed.
```
Comment (kimsauce) [inline docs/api-testing/environments.md:24]: Hi @LoryApiFortress , I pulled this from the legacy doc, which talked about creating a variable for the mastiff.apifortress.com domain. But I'm not sure if it's needed anymore.

**Comment by kimsauce** [comment]
@LoryApiFortress , thanks again for sending over the additions to this doc. I would like to add more screenshots, but wanted to get the written part to you for review first.

File: docs/api-testing/environments.md (hunk 1 above)
Comment (LoryApiFortress) [reply docs/api-testing/environments.md:24]: @kimsauce it's just an example how to parametrise a domain and creating variables for each part. You can leave it as is or replace it with any other domain

File: docs/api-testing/environments.md [hunk 2]
```diff
@@ -7,51 +7,63 @@ description: "Gain a wide range of options to mix and match your test settings w
...
+   * Import an existing environment by clicking **Import** > **Choose File** (accepted file types are Postman, .json, .csv).<br/><img src={useBaseUrl('img/api-fortress/2021/04/importEnv.png')} alt="create environment" width="500"/>
 
-3. Add the variable details and select **Confirm**:
-  <img src={useBaseUrl('img/api-fortress/2021/04/addDomainVar.png')} alt="variable in environment"/>
+ The resulting environment (along with its defined variables) will now appear each of your Tests as an option in the environment dropdown.
 
-  You can add multiple variables to each environment:
-  <img src={useBaseUrl('img/api-fortress/2021/04/result.png')} alt="result in environment"/>
+### From a Test
+1. Locate your desired Test from any of the following sections: **Tests**, **Compose**, **Schedule**, or **HTTP Client**. These are all of the places in the UI where you can run a Test.
```
Comment (LoryApiFortress) [inline docs/api-testing/environments.md:39]: @kimsauce **Schedule** is not a section where you can run test(s) and it does not have the dropdown for the Environments. Even though the override section serves as a substitute for the Environments it doesn't show you the list of available environments so you have to enter manually the variables that could be new variables not strickly related to an environment you have in the dropdown. 
**HTTP Client** is not the place where you can run a test, you can execute a call there and then generate a test

File: docs/api-testing/environments.md [hunk 3]
```diff
@@ -7,51 +7,63 @@ description: "Gain a wide range of options to mix and match your test settings w
...
+5. Click **Create variable**, enter **Key** and **Value** pairs, then click **Confirm**.
+6. When you're finished adding variables, click the **Close Environments Editor** icon.<br/><img src={useBaseUrl('img/api-fortress/2021/04/closeEditor.png')} alt="create environment" width="300"/>
 
-4. The resulting environment (along with the defined variables) is now selectable and appears in the environment section:
-  <img src={useBaseUrl('img/api-fortress/2021/04/result2.png')} alt="result in tests"/>
+This new environment will be selected by default in the dropdown.
 
-By activating a preset in the tests section, you will be able to hit a different domain in the current session without actually changing the tests.
+## Using Environments
+Once you've added created an environment and added the variables you'll need, you can then run your Test against that environment.
```
Comment (LoryApiFortress) [inline docs/api-testing/environments.md:49]: @kimsauce "Once you've added created an environment" added or created? :)

File: docs/api-testing/environments.md [hunk 4]
```diff
@@ -7,51 +7,63 @@ description: "Gain a wide range of options to mix and match your test settings w
...
 
-The same selection can be performed while creating a schedule to create specific runs hitting specific environments.
+1. Locate your desired Test from any of the following sections: **Tests**, **Compose**, **Schedule**, or **HTTP Client**. These are all of the places in the UI where you can run a Test.
```
Comment (LoryApiFortress) [inline docs/api-testing/environments.md:51]: @kimsauce same as the previous comment

**Review by LoryApiFortress — COMMENTED** [review]
@kimsauce required some changes here and there

**Comment by LoryApiFortress** [comment]
> @LoryApiFortress , thanks again for sending over the additions to this doc. I would like to add more screenshots, but wanted to get the written part to you for review first.

Thank you so much! Up to you about screenshots, add as much as you think are useful

File: docs/api-testing/environments.md (hunk 3 above)
Comment (kimsauce) [reply docs/api-testing/environments.md:49]: yikes!

**Review by LoryApiFortress — APPROVED** [review]

**Pull request merged**